Vagrantfile

```diff
@@ -13,6 +13,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         testvm.vm.box_url = "http://puppet-vagrant-boxes.puppetlabs.com/centos-65-i386-virtualbox-puppet.box"
 
         testvm.ssh.port = 2401
+        testvm.ssh.insert_key = false
```

Comment:
Worth adding a note that this should be removed again when issue x is fixed (link to github issue)?